### ISSUE TITLE
how to change the learning-rate？

### ISSUE THREAD

**Issue by W998K** [eriklindernoren/PyTorch-YOLOv3#388]
how to change the learning-rate？

**Comment by jaysonph** [comment]
@W998K 
Before training, you can simply set like:
> optimizer = torch.optim.Adam(model.parameters(), lr=0.0001)
Within training, I suggest you can try LR scheduler (very useful for training):
> optimizer = torch.optim.SGD(model.parameters(), lr=0.1, momentum=0.9)
> scheduler = ReduceLROnPlateau(optimizer, 'min')
> for epoch in range(10):
>     train(...)
>     val_loss = validate(...)
>     # Note that step should be called after validate()
>     scheduler.step(val_loss)

**Comment by Flova** [comment]
You can change the learning rate and learning rate reduction points in the `.cfg` file.
